lib/active_storage/service/cloudinary_service.rb

```diff
@@ -124,9 +133,12 @@ def download_chunk(key, range)
       instrument :download, key: key do
         req = Net::HTTP::Get.new(uri)
         range_end = case
-                    when range.end.nil? then ''
-                    when range.exclude_end? then range.end - 1
-                    else range.end
+                    when range.end.nil? then
```

Comment:
`then` is not necessary when using another line.

Reply:
My IDE made that change, reverted it.